### PR TITLE
Generate hotels table

### PR DESCRIPTION
**Pull request by Ronapj1991**

### CODE REVIEW

File: spec/factories/hotels.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  factory :hotel do
+    hotel_place_id { "MyString" }
+    name { "MyString" }
+    email { "MyString" }
+    phone { "MyString" }
+    website { "MyString" }
+    address { "MyString" }
+    image_url { "MyString" }
+    wheelchair { false }
+    stars { 1 }
```
Comment (dsinn) [inline spec/factories/hotels.rb]: I suggest using a combination of Faker, [SecureRandom.hex](https://ruby-doc.org/stdlib-3.0.2/libdoc/securerandom/rdoc/SecureRandom.html#module-SecureRandom-label-Examples) and [rand](https://ruby-doc.org/3.2/Kernel.html#method-i-rand) to fill these in.

File: app/models/hotel.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Hotel < ApplicationRecord
+    #belongs_to :city
```
Comment (dsinn) [inline app/models/hotel.rb:2]: Eh, the schema is going to have to change again once the City model is added. To keep things simple, I would advise waiting for City to be added. The design that we went over requires both models anyway.

File: spec/factories/hotels.rb (hunk 1 above)
Comment (Ronapj1991) [reply spec/factories/hotels.rb]: Added faker data 👍

File: spec/factories/attractions.rb [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-    name { "MyString" }
-    email { "MyString" }
-    phone { "MyString" }
-    website { "MyString" }
-    address { "MyText" }
-    image_url { "MyString" }
-    toilets { false }
-    wheelchair { false }
-    changing_table { false }
+    attraction_place_id { SecureRandom.hex(91) }
```
Comment (muginga72) [inline spec/factories/attractions.rb:3]: @Ronapj1991, what does the SecureRandom.hex(91)?

**Review by muginga72 — APPROVED** [review]

**Review by AigulY — APPROVED** [review]

File: app/models/hotel.rb (hunk 2 above)
Comment (Ronapj1991) [reply app/models/hotel.rb:2]: Cities table now merged

**Pull request merged**